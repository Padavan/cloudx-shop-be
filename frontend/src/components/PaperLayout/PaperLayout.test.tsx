import { render } from "@testing-library/react";
import PaperLayout from "./PaperLayout";

describe("<PaperLayout />", () => {
  it("should match to snapshot", () => {
    const component = render(
      <PaperLayout>
        <div />
      </PaperLayout>
    );

    expect(component.asFragment()).toMatchSnapshot();
  });
});
