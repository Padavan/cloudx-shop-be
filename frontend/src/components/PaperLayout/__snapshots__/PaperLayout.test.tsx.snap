// Vitest Snapshot v1

exports[`<PaperLayout /> > should match to snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-17ax1z0"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-53wyd7-MuiPaper-root"
    >
      <div />
    </div>
  </div>
</DocumentFragment>
`;
